packages/cli/package.json

```diff
@@ -17,7 +17,6 @@
     "chalk": "^2.4.2",
     "detect-indent": "^6.0.0",
     "find-up": "^4.1.0",
-    "fs-extra": "^8.1.0",
```

Comment:
ugh, I've missed it but we might also want to drop this: https://github.com/Thinkmill/manypkg/blob/d0f4d926040bca29b1f2dd9f366e60854fb16118/package.json#L32

Reply:
I'll add it to one of the next ones.

Wanted to add knip to catch stuff like this, but even though it's a dev dependency, it requires node 18.18. If we're fine with removing the engines field in the monrepo root?

Reply:
Hm, yee - the monorepo root doesn't have to maintain that node version compat. Although I guess having the `engines` there makes it easier to control stuff right now. But well, I'd still be OK with removing it from there and making sure it's in all other `package.json` of our publishable packages